src/configure/configurers/remoteGitHubWorkflowConfigurer.ts

```diff
@@ -169,6 +170,30 @@ export class RemoteGitHubWorkflowConfigurer extends LocalGitHubWorkflowConfigure
                         this.assets[asset.id] = await nodeVersionConverter.webAppRuntimeNodeVersionConverter(nodeVersion, armUri, this.azureSession);
                     }
                     break;
+                case "AzureCredentials:PublishProfile":
+                    {
+                        let resourceId = asset.inputs["resourceId"];
+                        let parsedResourceId = new ParsedAzureResourceId(resourceId);
+                        let subscriptionId = parsedResourceId.subscriptionId;
+
+                        this.assets[asset.id] = await vscode.window.withProgress(
+                            {
+                                location: vscode.ProgressLocation.Notification,
+                                title: utils.format(Messages.creatingAzureServiceConnection, subscriptionId)
+                            },
+                            async () => {
+                                try {
+                                    // find LCS of all azure resource params
+                                    let appServiceClient = new AppServiceClient(this.azureSession.credentials, this.azureSession.environment, this.azureSession.tenantId, subscriptionId);
+                                    return await appServiceClient.getWebAppPublishProfileXml(resourceId);
+                                }
+                                catch (error) {
+                                    telemetryHelper.logError(Layer, TracePoints.AzureServiceConnectionCreateFailure, error);
```

Comment:
do you want to rename the tracepoint to GetAzurePublishProfile

Reply:
updated